snap/snapcraft.yaml

```diff
@@ -37,6 +37,7 @@ parts:
       - wheel
       - python_magic
       - packaging
+      - boto3
```

Comment:
> @TrevorBenson lets change this to give it a specific version, I think the new version of the `boto3` may be incompatible with bionic, and hence causing issue. I rebuilt the snap with the new version, and didn't have this problem
> 
> ```
> - boto3==1.26.155
> ```

@TurboTurtle @pmoravec In the past, for Enterprise Linux systems when not also dealing with Debian packages, I have relied on Red Hat backporting to avoid breaking changes. At least for RHBAs and RHSAs, and fixed issues during an RHEA (rarely) when they had a breaking change. Is that how it's handled for sos or should I pin the boto3 version for RPMs as well to keep a similar logic on both distros?

Reply:
The difference here is that in this particular scenario it is installing these `python_packages` from pip.

If this was a deb package, then it wouldn't hav been a problem as it would have grabbed it from it's distro packages and would be compatible

Reply:
@arif-ali added commit 794619f to pin the version for the snap. I'll squash it once I hear back from the others if a similar change is wanted for the RPM.

Reply:
@TrevorBenson thanks for pinning the  `boto` version on snap, I opened #3481 so that we follow-up on this later on.